src/ServiceControl.Audit.AcceptanceTests/Auditing/When_processed_message_searched_by_body_content.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace ServiceControl.Audit.AcceptanceTests.Auditing
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Audit.Auditing.MessagesView;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTesting.Customization;
+    using NServiceBus.Settings;
+    using NUnit.Framework;
+    using TestSupport.EndpointTemplates;
+
+    class When_processed_message_searched_by_body_content : AcceptanceTest
+    {
+        [Test]
+        public async Task Should_be_found_when_fulltext_search_enabled()
+        {
+            // setting it even if it is the default
+            SetSettings = settings => settings.EnableFullTextSearchOnBodies = true;
+
+            var searchString = "42";
```

Comment:
Same potential issue as the test against the error instance. It's possible that any Guid could contain the string `42` and cause this test to be wonky

Reply:
Addressed